src/objects.js

```diff
@@ -1,41 +1,49 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) return true;
+  return false;
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) return true;
+  return false;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(newArray => newArray.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(thePerson => thePerson.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(theCars => theCars.manufacturer === "Honda");
 };
 
 const averageAge = people => {
-  // your code here
+  return people.reduce((a, b) => a + b.age, 0) / people.length;
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age,
+    introduce: hiya => {
```

Comment:
I love this parameter name. As it is another person's name, we could name it slightly better to be more descriptive of what it represents though. Eg `nameOfStranger`